treeherder/webapp/api/performance_data.py

```diff
@@ -889,6 +906,54 @@ def _get_signatures(self, repository_name, framework, interval, no_subtests):
         signatures = self._get_signatures_values(signatures)
         return signatures
 
+    def _create_graph_links(
+        self,
+        base_repo_name,
+        new_repo_name,
+        base_revision,
+        new_revision,
+        framework,
+        timeRange,
+        signature,
+    ):
+        params = {}
+        highlightedRevisionKey = 'highlightedRevisions'
+        seriesKey = 'series'
+        timeRangekey = 'timeRange'
+        seriesParams = None
+
+        if new_repo_name == base_repo_name:
+            # if repo for base and new are not the same then make diff
+            # series data one for each repo, else generate one
+            params['series'] = [new_repo_name, signature, 1, framework]
+        else:
+            params['series'] = [
+                [base_repo_name, signature, 1, framework],
+                [new_repo_name, signature, 1, framework],
+            ]
+
+        params[highlightedRevisionKey] = [base_revision, new_revision]
```

Comment:
```suggestion
        params[highlightedRevisionKey] = [short_base_revision, short_new_revision]
```
where `short_base_revision` is `base_revision[:12]`